docs/docs/how-to/firefly-iii/finances/reconcile.md

```diff
@@ -1,41 +1,53 @@
 # How to reconcile accounts
 
-Although more and more people use fancy online banking, a lot of users still receive paper bank statements in the mail, every week or every month.
+Firefly III has a "reconciliation" view which allows you to verify your firefly transactions exactly match your account statement.
+This is an easy way to have peace of mind that your bank or credit card company is (hopefully) honest. This can also help to resolve incorrect balances after importing a large set of historical transactions into Firefly III.
```

Comment:
I think the first sentence isn't helpful. Perhaps just say, peace of mind that your statements are correct?